docs/os-motivation.md

```diff
@@ -0,0 +1,20 @@
+---
+id: aragonos-motivation
+title: Motivations
+sidebar_label: Motivations
+---
+
+### Why we built aragonOS
+---
+
+aragonOS was born on our path to developing modular and upgradeable smart contracts to power decentralized organizations.
+
+As the project began maturing and we started abstracting more and more, we ended up encapsulating a lot of strong design decisions into a generic framework that could be used by any protocol or decentralized application wanting upgradeability.
+
+**Upgradeability** is really important when developing high-stakes systems in platforms that are still evolving fast. Done well, it can allow for bug fixes and improvements with very little disruption while not causing problems at the protocol level. As a history lesson, if *The DAO* had had an effective upgradeability mechanism, no hard fork would have been required to fix the problem. As a community, we could have avoided months of unproductive discussions, delays in the roadmap, and billions lost in the market capitalization of the protocol due to the fork's damage to network effects.
```

Comment:
could we provide a link to The Dao so it's clear what's being talked about?

Reply:
See 9369fbe